src/token.ts

```diff
@@ -54,7 +54,7 @@ export const enum Token {
   RightBrace   = 15 | IsAutoSemicolon | IsClassField, // }
   RightParen   = 16, // )
   Semicolon    = 17 | IsAutoSemicolon | IsClassField, // ;
-  Comma        = 18 | IsClassField, // ,
+  Comma        = 18, // ,
```

Comment:
@KFlash was this just a fat finger mistake? I want to make sure I didn't miss anything.

Reply:
Dunno. Seems like it. Just remove it. Shouldn't be valid